ci/jenkins/test.sh

```diff
@@ -666,7 +666,7 @@ function run_e2e {
     if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
     elif [[ $TESTBED_TYPE == "kind" ]]; then
-        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider kind -timeout=100m --prometheus
+        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider kind --kind.kubeconfig ${KUBECONFIG_PATH} -timeout=100m --prometheus
     elif [[ $TESTBED_TYPE == "kind-flexible-ipam" ]]; then
```

Comment:
I think you also need to pass `--kind.kubeconfig ${KUBECONFIG_PATH}` for this TESTBED_TYPE.

Reply:
currently we are not supporting multiple flexible-ipam test run on a single vm.

Reply:
Even we didn't support multiple flexible-ipam test run on a single vm, you still need to assign the value explicitly, otherwise, the default config `path.Join(homedir, ".kube", "config")` will be used, which will be different with the one assigned in CI job: `${{PWD}}/.kube/config`. e.g. `./ci/jenkins/test.sh --testcase e2e --registry ${{DOCKER_REGISTRY}} --kubeconfig ${{PWD}}/.kube/config --testbed-type "kind-flexible-ipam" --kind-cluster-name "${{FULL_CLUSTER_NAME}}"`.